crates/ra_ide_api/src/parent_module.rs

```diff
@@ -15,10 +19,12 @@ pub(crate) fn parent_module(db: &RootDatabase, position: FilePosition) -> Vec<Na
 
 /// Returns `Vec` for the same reason as `parent_module`
 pub(crate) fn crate_for(db: &RootDatabase, file_id: FileId) -> Vec<CrateId> {
-    let module = match hir::source_binder::module_from_file_id(db, file_id) {
-        Some(it) => it,
-        None => return Vec::new(),
-    };
+    let src = hir::ModuleSource::from_file_id(db, file_id);
+    let module =
+        match hir::Module::from_definition(db, hir::Source { file_id: file_id.into(), ast: src }) {
+            Some(it) => it,
+            None => return Vec::new(),
+        };
```

Comment:
Hm, all these `Module::from_definition` seem pretty verbose... I wonder if we should add (in a spearate PR) an helper method to get a module to `RootDatabase`...